plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx

```diff
@@ -76,7 +85,51 @@ function ProductCardFooter( props: { product: Product } ) {
 			return __( 'Free plan available', 'woocommerce' );
 		}
 
-		return currencySymbol + product.price;
+		return getCurrencySymbol( product.currency ) + product.price;
```

Comment:
Part 2 of my suggestion: use the format string to format the currency -

```suggestion
		return sprintf( getCurrencyFormat( product.currency ), product.price );
```